scripts/shared/build_image.sh

```diff
@@ -22,19 +22,24 @@ dockerfile="${FLAGS_dockerfile}"
 buildargs="${FLAGS_buildargs}"
 platform="${FLAGS_platform}"
 hashfile="${FLAGS_hash}"
+ocifile="${FLAGS_oci}"
 [[ "${FLAGS_cache}" = "${FLAGS_TRUE}" ]] && cache=true || cache=false
 
 [[ -n "${image}" ]] || { echo "The image to build must be specified!"; exit 1; }
 [[ -n "${dockerfile}" ]] || { echo "The dockerfile to build from must be specified!"; exit 1; }
 [[ -n "${hashfile}" ]] || { echo "The file to write the hash to must be specified!"; exit 1; }
+if [[ "${platform}" =~ , && -z "${ocifile}" ]]; then
```

Comment:
What does regex-comparing to a comma (`=~ ,`) do? Is it something like a parameter expansion and nil check in one?

Reply:
We can also ask/answer this on Stackoverflow if you want, with how I currently know to describe it I don't see any existing question.

Reply:
There’s probably an SO/SE covering this somewhere already! `=~` is a regex comparison, it allows substring matches; so this is looking for commas anywhere in `"${platform}"`, which indicates that multiple platforms have been requested.

Reply:
> This is already covered on SE

Thanks! (I deleted my comment a few moments after posting, before seeing your reply, for fear it was there and I was just searching poorly)

Reply:
Ah okay, yeah I read exactly that SE post and already knew `~=`, but I was over-thinking the comma. I thought you were using it like https://unix.stackexchange.com/questions/500274/what-do-the-commas-in-var-mean, not a literal.